internal/report/report.go

```diff
@@ -309,7 +309,10 @@ func printTopProto(w io.Writer, rpt *Report) error {
 	}
 	functionMap := make(functionMap)
```

Comment:
The coverage data says this code block is not covered by tests. Is it a bug with the coverage data or there is something in it?

Reply:
I'm incline to think there is a big with the coverage data.
Several of functions for printing the output format aren't covered, and I've added are tests that go from passing to failing when this function is modified.